haystack/utils/deepsetcloud.py

```diff
@@ -1204,6 +1204,44 @@ def update_eval_run(
         )
         return response.json()["data"]
 
+    def get_eval_run_results(
+        self, eval_run_name: str, workspace: Optional[str] = None, headers: dict = None
+    ) -> Dict[str, Any]:
+        """
+        Collects and returns the predictions of an evaluation run.
+
+        :param eval_run_name: The name of the evaluation run to update.
+        :param workspace: Specifies the name of the workspace on deepset Cloud.
+                          If None, the FileClient's default workspace is used.
+        :param headers: Headers to pass to API call
+        """
+
+        response = self.get_eval_run(eval_run_name, workspace, headers)
+        predictions_per_node = {}
+        for eval_result in response["eval_results"]:
+            predictions_per_node[eval_result["node_name"]] = self.get_eval_run_prediction(
+                eval_run_name=eval_run_name, node_name=eval_result["node_name"], workspace=workspace, headers=headers
+            )
+
+        return predictions_per_node
+
+    def get_eval_run_prediction(
+        self, eval_run_name: str, node_name: str, workspace: Optional[str] = None, headers: Optional[dict] = None
+    ) -> List[Dict[str, Any]]:
+        """
+        Gets predictions for an evaluation run and node name.
+
+        :param eval_run_name: The name of the evaluation run to update.
```

Comment:
Again, is it an evaluation run to update or an evaluation run whose results we want to fetch?